LICENSE

```diff
@@ -0,0 +1,27 @@
+mit License
+
+  Copyright (c) 2023 Robert Kanyoro and IWU John
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+the above copyright notice and this permission notice shall be included in all
+  copies or substantial portions of the software.
+  the software is provided "AS IS",
+  without warranty of any kind,
+  express or
+  implied,
+  including but not limited to the warranties of merchantability,
+  fitness for a particular purpose and noninfringement. in no event shall the
+  authors or copyright holders be liable for any claim,
+  damages or other
+  liability,
+  whether in an action of contract,
+  tort or otherwise,
+  arising from,
+  out of or in connection with the software or the use or other dealings in the
+  software.;LicenseCopyright2023RobertKanyoroandIWUJohnPermissionisherebygranted,freeofcharge,toanypersonobtainingacopyofthissoftwareandassociateddocumentationfilesthetodealintheSoftwarewithoutrestriction,includingwithoutlimitationtherightstouse,copy,modify,merge,publish,distribute,sublicense,and/orsellcopiesoftheSoftware,andtopermitpersonstowhomtheSoftwareisfurnishedtodoso,subjecttothefollowingconditionsLicenseCopyright2023RobertKanyoroPermissionisherebygranted,freeofcharge,toanypersonobtainingacopyofthissoftwareandassociateddocumentationfilesthetodealintheSoftwarewithoutrestriction,includingwithoutlimitationtherightstouse,copy,modify,merge,publish,distribute,sublicense,and/orsellcopiesoftheSoftware,andtopermitpersonstowhomtheSoftwareisfurnishedtodoso,subjecttothefollowingconditions
```

Comment:
- [ ] Your is almost ready for approval because it persists so well now however you seem to have forgotten to implement the formatting issues that the previous code reviewer raised. See how it looks :point_down: 
![image](https://github.com/Kanyorok/spacex-project/assets/99715640/0c76a03c-dd96-4185-8e92-fc54af3b56f4)

This is not really impressive for recruiters or reviewers to see in your code base. You are a professional software developer and that is what your profile, works, and code must never fail to show :wink: